nusight2/src/client/components/app/style.module.css

```diff
@@ -1,11 +1,3 @@
-@import url('https://fonts.googleapis.com/css?family=Open+Sans:300,400,500');
-
-html,
-body {
-  margin: 0;
-  padding: 0;
-}
-
 .app {
   font: 14px 'Open Sans', Helvetica, Arial, sans-serif;
   min-height: 100vh;
```

Comment:
This is for the `font-weight` line below, can't comment directly on that.

Would it make sense to remove that (in a separate PR if necessary)? The 300 font weight reduces the text contrast a fair bit, especially for small sized text. Another issue is that it's not in Storybook, so the components are developed with a normal 400 weight but get displayed at 300 in NUsight.

![Screenshot of before and after](https://user-images.githubusercontent.com/5924865/190076786-2a12f9ac-e735-4b2f-927c-051944a4098a.png)